config/routes.rb

```diff
@@ -1,5 +1,16 @@
 Rails.application.routes.draw do
-  get 'users/hello'
+  get 'comments/index'
+  get 'likes/index'
+  get 'posts/index'
+  root to: 'users#index'
+
+  resources :users, only: %i[index show] do
+    resources :posts, only: %i[index new ] do
+      resources :likes, only: %i[new ]
+    resources :comments, only: %i[new ]
+    end
```

Comment:
- [ ] It is good practice to only expose needed routes. We don't really need to expose all these routes yet. Kindly remove unnecessary routes. We can have something like so:

  ```
  root to: 'users#index'

  resources :users, only: %i[index show] do
    resources :posts, only: %i[index new ]
  end
  ```

Reply:
This issue is well resolved! ✔️